samples/cpp/tutorial_code/Histograms_Matching/MatchTemplate_Demo.cpp

```diff
@@ -4,11 +4,9 @@
  * @author OpenCV team
  */
 
-#include "opencv2/imgcodecs.hpp"
 #include "opencv2/highgui/highgui.hpp"
 #include "opencv2/imgproc/imgproc.hpp"
```

Comment:
Should you not replace the headers above also to

```
 #include "opencv2/highgui.hpp"
 #include "opencv2/imgproc.hpp"
```

to be compliant to OpenCV 3 headers